spec/src/main/asciidoc/ch04-query-language.adoc

```diff
@@ -274,14 +274,21 @@ from_clause ::=
     FROM identification_variable_declaration
         {, {identification_variable_declaration | collection_member_declaration}}*
 
-identification_variable_declaration ::= range_variable_declaration {join | fetch_join}*
+identification_variable_declaration ::= range_variable_declaration join*
 
 range_variable_declaration ::= entity_name [AS] identification_variable
 
-join ::= join_spec join_association_path_expression [AS] identification_variable [join_condition]
+join ::= range_join | path_join | fetch_join
```

Comment:
This change would allow fetch joins in a subquery, which is obviously wrong. See `subselect_identification_variable_declaration`, which uses the `join` rule.

Reply:
Ah, OK, thanks for catching that 👍